.readthedocs.yaml

```diff
@@ -27,6 +27,7 @@ build:
     # Altered PDF build and upload job
     # This also runs on PR builds, but does not upload the PDF
     post_build:
+      - export BUILD_IDAKLU=OFF
```

Comment:
Since `wheel.cmake` is now `false` by default, can the RTD builds work without setting this environment variable?

Reply:
I will test it and let you know

Reply:
Just tested the RTD builds, looks like it is working fine: [LINK](https://readthedocs.org/projects/migrate-test/builds/25225826/)